ts/a11y/semantic-enrich.ts

```diff
@@ -240,12 +240,17 @@ export function EnrichedMathDocumentMixin<N, T, D, B extends MathDocumentConstru
     public static OPTIONS: OptionList = {
       ...BaseDocument.OPTIONS,
       enableEnrichment: true,
-      enrichSpeech: 'none',                   // or 'shallow', or 'deep'
       renderActions: expandable({
         ...BaseDocument.OPTIONS.renderActions,
         enrich:       [STATE.ENRICHED],
         attachSpeech: [STATE.ATTACHSPEECH]
-      })
+      }),
+      sre: expandable({
+        speech: 'none',                    // by default no speech is included
+        domain: 'mathspeak',               // speech rules domain
+        style: 'default',                  // speech rules style
+        locale: 'en'                       // switch the locale
+      }),
```

Comment:
See the discussion in `explorer.ts` about whether this should be expandable or not.